.changes/1.3.0.md

```diff
@@ -0,0 +1,4 @@
+## dbt-snowflake 1.2.0 - July 12, 2022
+
+### Fixes
+- Add unique\_id field to docs generation test catalogs; a follow-on PR to core PR. ([#4168](https://github.com/dbt-labs/dbt-core/pull/4618))
```

Comment:
Are these 1.2.0 changes?  If so they should not be in this file.

Reply:
This file should also be generated via a changie command so the file name indicates it's a beta release.  It also allows all the yaml files to end up in the `1.3.0` folder so that they get included in the `1.3.0` final release.

`changie batch 1.3.0 --move-dir '1.3.0' --prerelease 'b1'`

Reply:
deleted current file, removed unneeded, 1.1.0 ref, rand suggested command

Reply:
@McKnight-42 you still need to run 

`changie batch 1.3.0 --move-dir '1.3.0' --prerelease 'b1'`

since the beta is already released.  It will generate the `1.0.3-b1.md` file and move all the yaml files from `/unreleased` and to `/1.3.0`.

Reply:
Have run changie batch command. Curious to what the process is here for releases when do we typically trigger the new batch command in normal flow during release?